.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.9
+      # Install dependencies
+      - name: Install Dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+
+      # Run tests
+      - name: Run Tests
+        run: make model-test
```

yamllint (1.29.0-1)

[error] no new line character at the end of file

(new-line-at-end-of-file)